StyleCop.Analyzers/StyleCop.Analyzers.CodeGeneration/SyntaxLightupGenerator.cs

```diff
@@ -1393,5 +1441,20 @@ public string GetAccessorResultType(SyntaxData syntaxData)
                 return null;
             }
         }
+
+        private record CompilationData(ImmutableDictionary<string, ExistingTypeData> ExistingTypes)
+        {
+        }
+
+        private record ExistingTypeData(string TypeName, ImmutableArray<string> MemberNames)
```

Comment:
@chsienki is there a straightforward way to have a properly equatable list of strings in a record, without having to implement custom equality?